### ISSUE TITLE
No longer works with download 5.0.0

### ISSUE THREAD

**Issue by kevin-smets** [kevva/download-status#13]
The usage is broken on download 5.0.0. Could you add this to the readme maybe or the installation instructions? Are there any plans on fixing this?

**Comment by zce** [comment]
https://github.com/sindresorhus/got#streams

**Comment by kevva** [comment]
Yeah, still not decided if I should deprecate this or make it work with `5.x.x` yet.

**Comment by asgoth** [comment]
@kevva With your other npm module [decompress](https://github.com/kevva/decompress), you provide the option of [plugins](https://github.com/kevva/decompress#plugins). Isn't this a bit similar?

**Comment by kevva** [comment]
Check out https://github.com/kevva/download/issues/106#issuecomment-243071351 if you want progress with `5.0.0`.
